### ISSUE TITLE
Wrong return types in php code

### ISSUE THREAD

**Issue by SonicGD** [grpc/grpc#23736]
Hello. 

### What version of gRPC and what language are you using?
gRPC 1.30.2. 
PHP 7.3

### What operating system (Linux, Windows,...) and version?
Debian 10.5

### What did you do?
Today i've updated gprc-php-plugin and regenerated php code from my protofiles.After that PhpStorm and phpstan started to report type errors. After some research i've found this pr - #22671. It adds `return` phpdoc attribute to methods in generated client code. For example, let's generate `Math.proto`

Before this pr generated code for `Div` looked like this:
```php
/**
 * Div divides DivArgs.dividend by DivArgs.divisor and returns the quotient
 * and remainder.
 * @param \Math\DivArgs $argument input argument
 * @param array $metadata metadata
 * @param array $options call options
 */
public function Div(\Math\DivArgs $argument,
  $metadata = [], $options = []) {
    return $this->_simpleRequest('/math.Math/Div',
    $argument,
    ['\Math\DivReply', 'decode'],
    $metadata, $options);
}
```

After it looks like this:
```php
/**
 * Div divides DivArgs.dividend by DivArgs.divisor and returns the quotient
 * and remainder.
 * @param \Math\DivArgs $argument input argument
 * @param array $metadata metadata
 * @param array $options call options
 * @return \Math\DivReply
 */
public function Div(\Math\DivArgs $argument,
  $metadata = [], $options = []) {
    return $this->_simpleRequest('/math.Math/Div',
    $argument,
    ['\Math\DivReply', 'decode'],
    $metadata, $options);
}
```

Now, in code
```php
<?php

use Grpc\ChannelCredentials;

require_once 'vendor/autoload.php';

$client = new \Math\MathClient('localhost', [
    'credentials' => ChannelCredentials::createInsecure()
]);
$result = $client->Div(new \Math\DivArgs());

echo get_class($result) . PHP_EOL;

[$response, $status] = $result->wait();
```

IDE thinks that variable `$result` is of type `\Math\DivReply`:
![image](https://user-images.githubusercontent.com/1222013/89334691-582aa780-d6b0-11ea-89a9-eff8e83bfbde.png)

But it is not, it is of type `Grpc\UnaryCall`:
![image](https://user-images.githubusercontent.com/1222013/89334745-72fd1c00-d6b0-11ea-9f7b-5e20eb17e7f9.png)

And this is correct. But now, if we try to call `wait()` on result - there is ide error:
![image](https://user-images.githubusercontent.com/1222013/89334997-d8510d00-d6b0-11ea-9556-20ec8aa19cd5.png)

### What did you expect to see?
I think #22671 should be reverted or further improved. One way to improve is to replace 
```php
 * @return \Math\DivReply
```
with
```php
 * @return \Grpc\UnaryCall
```
or, to preserve result type, with additional comment:
```php
 * @return \Grpc\UnaryCall call with response type \Math\DivReply
```

I can provide PR if needed. Thanks.

**Comment by stanley-cheung** [comment]
Fixed via #23907.

**Comment by SonicGD** [comment]
Thanks!
